app/schema/mutations/application/createNewFormData.ts

```diff
@@ -7,7 +7,14 @@ const mutation = graphql`
   mutation createNewFormDataMutation($input: CreateNewFormDataInput!) {
     createNewFormData(input: $input) {
       formData {
+        id
```

Comment:
Do we still need this change now that we're not creating a new form data?

Reply:
No I'll remove it thanks!